### PR TITLE
add custom hmda message for populating institutions

### LINKED ISSUE #82: Pull some message from properties
It might be better to pull some content from [`messages_en.properties`](https://github.com/cfpb/hmda-platform-auth/tree/20773fbbcb475b0c9f985a303e1758560a9d6bd4/keycloak/themes/hmda/login/messages). For example:

- [this](https://github.com/cfpb/hmda-platform-auth/blob/20773fbbcb475b0c9f985a303e1758560a9d6bd4/keycloak/themes/hmda/login/register.ftl#L31)
- also used [here](https://github.com/cfpb/hmda-platform-auth/blob/20773fbbcb475b0c9f985a303e1758560a9d6bd4/keycloak/themes/hmda/login/register.ftl#L165)
- and maybe [this one](https://github.com/cfpb/hmda-platform-auth/blob/20773fbbcb475b0c9f985a303e1758560a9d6bd4/keycloak/themes/hmda/login/register.ftl#L135)
- and [this one](https://github.com/cfpb/hmda-platform-auth/blob/20773fbbcb475b0c9f985a303e1758560a9d6bd4/keycloak/themes/hmda/login/register.ftl#L135)

### LINKED ISSUE #87: Update instructions for registering for multiple institutions
Two scenarios where users need to register for multiple institutions: 
1) To register for multiple institutions that auto-populate based on the user’s email address
2) To register for multiple institutions that will not auto-populate based on the user’s email address. 

Under the current “select your institutions” instructions on the registration screen, include:
“To register as a filer for more than one institution, you may select multiple institutions from the list of available institutions based on your email domain. If one or more institutions is not available in the list based on your email domain, contact [support email address] for assistance with registering for the additional institution(s).”

### PR DESCRIPTION
**Pull request by awolfe76**
Closes #82 
Closes #87 

# Screenshot for #87
newmessage

### CODE REVIEW

File: keycloak/themes/hmda/login/messages/messages_en.properties [hunk 1]
```diff
@@ -24,3 +24,7 @@ invalidPasswordNotUsernameMessage=&#8226; Invalid password: must not be equal to
 invalidPasswordRegexPatternMessage=&#8226; Invalid password: fails to match regex pattern(s).
 invalidPasswordHistoryMessage=&#8226; Invalid password: must not be equal to any of last {0} passwords.
 invalidPasswordGenericMessage=&#8226; Invalid password: new password doesn''t match password policies
+
+# custom for hmda
+# DO NOT COMMIT UPDATES TO OR REMOVE THE "support@localhost.localdomain"
+hmdaEnterEmailAddress=To register as a filer for more than one institution, you may select multiple institutions from the list of available institutions based on your email domain. If one or more institutions is not available in the list based on your email domain, contact support@localhost.localdomain for assistance with registering for the additional institution(s).
```
Comment (hkeeler) [inline keycloak/themes/hmda/login/messages/messages_en.properties]: I _think_ we could use a template variable for that email address, replacing `support@localhost.localdomain` w/ `{0}`, and then in the template use ${msg("hmdaEnterEmailAddress", (properties.supportEmailTo!''))}.  Then you can even get rid of the find/replace bit too. :)

Comment (awolfe76) [inline keycloak/themes/hmda/login/messages/messages_en.properties]: Does that require some work on your end?

Comment (hkeeler) [inline keycloak/themes/hmda/login/messages/messages_en.properties]: In theoryland, it should just work.  All attributes in [`hmda/login/theme.properties`](https://github.com/cfpb/hmda-platform-auth/blob/master/keycloak/themes/hmda/login/theme.properties) should be available in the templates with the `properties` object.

Unfortunately, I don't think there really is a guide.  I think thing `msg` function is a freemark util cooked up by Keycloak.  But, there are example of passing values into templatized attributes:

* [`hmda/login/register.ftl:4`](https://github.com/cfpb/hmda-platform-auth/blob/master/keycloak/themes/hmda/login/register.ftl#L4)

Comment (awolfe76) [inline keycloak/themes/hmda/login/messages/messages_en.properties]: Updated in https://github.com/cfpb/hmda-platform-auth/pull/112/commits/0ecbade8e266f1e03003859af4375ae297611683

**Pull request merged**